idea/frontend/src/features/sails/api/requests.ts

```diff
@@ -1,13 +1,30 @@
 import { HexString } from '@gear-js/api';
+import ky from 'ky';
 
 import { PaginationResponse } from '@/api';
-import { INDEXER_RPC_SERVICE, rpcService } from '@/shared/services/rpcService';
+import { INDEXER_RPC_SERVICE } from '@/shared/services/rpcService';
+import { META_STORAGE_API_URL } from '@/shared/config';
 
 import { EventType, GetEventsParameters } from './types';
 import { METHOD } from './consts';
 
-const addIdl = (codeId: HexString, data: string) => rpcService.callRPC(METHOD.ADD_SAILS, { codeId, data });
-const getIdl = (codeId: HexString) => rpcService.callRPC<string>(METHOD.GET_SAILS, { codeId });
+type GetIdlResult = {
+  codeId: string;
+  data: string;
+};
```

Comment:
```suggestion
type GetIdlResponse = {
  codeId: string;
  data: string;
};
```

Also, let's move it to `./types`.